From 6003a8875915138fa2f7fe78efb41318341c2cf1 Mon Sep 17 00:00:00 2001
From: Ilya Maximets <i.maximets@ovn.org>
Date: Mon, 3 Jun 2024 13:12:04 +0200
Subject: [PATCH] tests: Fix non-portable plus match in python vlog test.

'\+' as a one-or-more match is a GNU extension and it doesn't work
in BSD sed.  This makes the python vlog test to fail on FreeBSD 14
that recently got python 3.11 in CirrusCI:

 |  --- -   2024-06-03 10:42:26.363566000 +0000
 |  +++ /tmp/cirrus-ci-build/tests/testsuite.dir/at-groups/2541/stdout
 |  @@ -7,31 +7,37 @@
 |   Traceback (most recent call last):
 |     File <name>, line <number>, in main
 |       assert fail
 |  +           ^^^^

Remove the '\+' match to make the line removal work.  It doesn't do
much for us as we would remove the same lines either way.

This change makes CirrusCI green again.

Fixes: 9185793e7543 ("tests: Fix compatibility issue with Python 3.13 in vlog.at.")
Acked-by: Eelco Chaudron <echaudro@redhat.com>
Signed-off-by: Ilya Maximets <i.maximets@ovn.org>
---
 tests/vlog.at | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/vlog.at b/tests/vlog.at
index efe91479a..2768c0740 100644
--- a/tests/vlog.at
+++ b/tests/vlog.at
@@ -8,7 +8,7 @@ AT_CHECK([$PYTHON3 $srcdir/test-vlog.py --log-file log_file \
 
 AT_CHECK([sed -e 's/.*-.*-.*T..:..:..Z |//' \
 -e 's/File ".*", line [[0-9]][[0-9]]*,/File <name>, line <number>,/' \
--e '/\^\+/d' \
+-e '/\^/d' \
 stderr_log], [0], [dnl
   0  | module_0 | EMER | emergency
   1  | module_0 | ERR | error
-- 
2.39.5

